content/efcore/getting-started/README.md

```diff
@@ -0,0 +1,7 @@
+# Getting Started with Entity Framework Core
+
+## Who Should Read This?
+
+This tutorial is meant for those new to Entity Framework (EF) Core, but if you already have an understanding of EF Core, consider this a refresher course. More advanced functionality is covered in the [docs](https://docs.efproject.net/en/latest/) .
```

Comment:
Could we remove the "if you already have an understanding..." part? The other tutorials don't say it, and I don't think we need to either. Experienced coders will understand they can use it as a refresher course without us needing to say it since we say it is for beginners.